ansible_collections/arista/avd/molecule/eos_validate_state/inventory/group_vars/DC1.yml

```diff
@@ -117,14 +117,17 @@ l2leaf:
   defaults:
     platform: vEOS-lab
     spanning_tree_mode: mstp
-
+    inband_mgmt_subnet: 172.21.110.0/24
+    inband_mgmt_description: L2LEAF_INBAND_MGMT
+    inband_mgmt_vlan_name: L2LEAF_INBAND_MGMT
+    inband_mgmt_vlan: 4085
+    inband_mgmt_vrf: MGMT
```

Comment:
Can we do same for l3 leaf switches, i was trying it but then am not getting inband_mgmt_vlan 4085 in structured  config.

Reply:
The `eos_designs` data model is for L2 switches --> https://avd.sh/en/stable/roles/eos_designs/docs/input-variables.html#node-type-inband-management